files/default/lib/bumpzone.rb

```diff
@@ -0,0 +1,110 @@
+#!/opt/chef/embedded/bin/ruby
+require 'git'
+require 'json'
+require 'octokit'
+
+# Library to bump DNS zones
+class BumpZone
+  @commit_msg = "\n\n"
+  @warning_msg = "Warning:\n\n"
+  @warning = false
+  ONELINE_REGEX = /(^[\w\d\.\-]+\s+[\d]+\s+IN\s+SOA.*[\w\d\.\-]+\s+)(\d{10})(\s+[\d]+\s+[\d]+\s+[\d]+\s+[\d].*)/
+  MULTILINE_REGEX =
+    /(^[\w\d\.\-]+\s+IN\s+SOA\s+[\w\d\.\-]+\s+[\w\d\.\-]+\s+\((?:\s*;(?:[\s\w\d]*)?)?(?:\s*)?\n\s+)(\d{10})\
+(.*\n.*\n.*\n.*\n.*\n\s+\))/
```

Comment:
My latest update to this produces a new warning when run: ``warning: regular expression has redundant nested repeat operator '*'``

I can't figure out what's causing this, can someone please take a look and let me know? Feel free to use this [site](http://regexr.com/3fcmg) to troubleshoot and ensure it works properly too.